pkg/gits/helpers.go

```diff
@@ -713,13 +716,15 @@ func ForkAndPullRepo(gitURL string, dir string, baseRef string, branchName strin
 	return dir, baseRef, upstreamInfo, forkInfo, nil
 }
 
-func configureJxAsGitCredentialHelper(dir string, gitter Gitter) error {
+func configureJxAsGitCredentialHelper(dir string, gitter Gitter, repoOwner string) error {
 	// configure jx as git credential helper for this repo
 	jxProcessBinary, err := os.Executable()
 	if err != nil {
 		return errors.Wrapf(err, "unable to determine jx binary location")
 	}
-	return gitter.Config(dir, "--local", "credential.helper", fmt.Sprintf("%s step git credentials --credential-helper", jxProcessBinary))
+	config := []string{"--local", "credential.helper", fmt.Sprintf("%s step git credentials --credential-helper --repo-owner %s", jxProcessBinary, repoOwner)}
```

Comment:
+1, definitely feels like the smallest possible change to make this work for your use case which at the same time is not completely irrational for the other users.